src/borg/remote.py

```diff
@@ -312,6 +313,10 @@ def negotiate(self, client_data):
         # clients since 1.1.0b3 use a dict as client_data
         if isinstance(client_data, dict):
             self.client_version = client_data[b'client_version']
+            if client_data.get(b'client_supports_log_v3', False):
```

Comment:
maybe also add a TODO about removing this client_supports_log_v3 check in 1.1rc?
if we get into this block, we have a borg 1.1 at the other end.